api/protos/bootstrap/v1/bootstrap.proto

```diff
@@ -51,6 +51,13 @@ message Upstream {
     // Timeout in seconds for upstream connection stream.
     // If unset defaults to no timeout.
     int64 timeout = 3;
+
+    // Jitter in seconds for upstream connection. Used with timeout to reset connections without overloading
+    // the upstream server. Jitter is the lower tolerance for random variation in the timeout. e.g. timeout=15,
+    // jitter=5 -> connectionTimeout is rand Int 10 and 15.
```

Comment:
It might make more sense if jitter is the upper tolerance since timeouts should generally be respected. If the connection isn't as long-lived as users expect, we may run into unexpected behavior from the client.